fitbenchmarking/fitbenchmarking
# Support use of BUMPS with Hellinger cost functions

#### Description of Work

Fixes #1101 


#### Testing Instructions

1. Run using bumps and the hellinger cost function
2. Check the results look similar to those with other minimizers
3. Repeat the test with other types of cost function

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
